sections/popular-products.liquid

```diff
@@ -0,0 +1,139 @@
+{{ 'popular-products.css' | asset_url | stylesheet_tag }}
+<h3 class="title title--popular-products">Lo más popular</h3>
```

Comment:
Remember: content must be customizable

From #3:
<img width="808" alt="image" src="https://github.com/Luis-Esquivia/analyticstheme-shopify/assets/16655413/58bc2c42-c406-49fa-8d43-e2ad7894cf2d">


```suggestion
<h3 class="title title--popular-products">{{ section.settings.title }}</h3>
```